frontend/packages/akt/src/pages/PrivacyPolicyPage.tsx

```diff
@@ -0,0 +1,187 @@
+import { Grid } from '@mui/material';
+import { FC, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import { ExtLink, H1, H2, H3, HeaderSeparator, Text } from 'shared/components';
+import { CommonUtils } from 'shared/utils';
+
+import { usePrivacyTranslation } from 'configs/i18n';
+
+type BulletListProps = {
+  translations: Array<string>;
+};
+
+const BulletList: FC<BulletListProps> = ({ translations }) => {
+  const translatePrivacy = usePrivacyTranslation();
+
+  return (
+    <ul>
+      {translations.map((translation, i) => (
+        <li key={i}>
+          <Text>{translatePrivacy(translation)}</Text>
+        </li>
```

Comment:
Tyylit taitaa korjaantua kun tekee niin kuin saavutettavuussivulla ja laittaa `<Text>...</Text>`-blokin `<li>...</li>` ympärille eikä toisin päin.